apps/userdashboard/views.py

```diff
@@ -45,9 +44,6 @@ def projects(self):
         projects = Project.objects.filter(
             follow__creator=self.request.user, follow__enabled=True
         )
-        for project in projects:
-            project.latest_comments_num = helpers.get_num_latest_comments(project)
```

Comment:
I couldn't find any place where this was used so I removed it. Or did I miss this somehow?

Reply:
I think we maybe used to use it to say so many comments in the last week but don't think we use it anymore but maybe it can also be removed from helpers.py

Reply:
I will add it back in then!